lib/app/modules/argonaut/argonaut_patient_read_only_sequence.rb

```diff
@@ -99,7 +99,7 @@ class ArgonautPatientReadOnlySequence < SequenceBase
       #   metadata {
       #     id '00'
       #     optional
-      #     desc %(
+      #     description %(
```

Comment:
Should this just be deleted or are we keeping it around for something?

Reply:
I don't know anything about these commented out tests, but it seems likely that they should go. I'm not going to hold up this PR over it, though, because the deprecation warnings are really annoying.